connector/subgraph/subgraph.yaml

```diff
@@ -37,13 +37,48 @@ dataSources:
         - name: MiniMeTokenFactory
           file: ./abis/MiniMeTokenFactory.json
       eventHandlers:
-        - event: NewAppProxy(address,bool,bytes32)
-          handler: handleNewAppProxy
-      file: ./src/aragon/mappings/Kernel.ts
-# /Organizations
-# Organization factories
+        - event: DeployDAO(address)
+          handler: handleDeployDAO
+      file: ./src/aragon/mappings/DAOFactory.ts
 # /Organization factories
 # Organization templates
+  # DAOTemplate
+  - kind: ethereum/contract
+    name: karma-template.aragonpm.eth@1.0.0
+    network: xdai
+    source:
+      abi: DAOTemplate
+      address: '0x5c1ed258ad3716c5ec4c6818dc786a6f6b05a7f0'
+      startBlock: 10543855
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - NoEnt
+      abis:
+        - name: DAOFactory
+          file: ./abis/DAOFactory.json
+        - name: DAOKit
+          file: ./abis/DAOKit.json
+        - name: DAOTemplate
+          file: ./abis/DAOTemplate.json
+        - name: Kernel
+          file: ./abis/Kernel.json
+        - name: MiniMeToken
+          file: ./abis/MiniMeToken.json
+        - name: MiniMeTokenFactory
+          file: ./abis/MiniMeTokenFactory.json
+      eventHandlers:
+        - event: DeployDao(address)
+          handler: handleDeployDao
+        - event: SetupDao(address)
+          handler: handleSetupDao
+        - event: DeployToken(address)
+          handler: handleDeployToken
+        - event: InstalledApp(address,bytes32)
+          handler: handleInstalledApp
+      file: ./src/aragon/mappings/DAOTemplate.ts
```

Comment:
Was wondering if we should add this file `subgraph.yaml` to the .gitignore since it would be generated from the `subgraph.template.yaml`?

Reply:
Yes, let's do that. I'll include that change and will merge the PR 👍